### DISCUSSION TITLE
Error "Python int too large to convert to C long"

### DISCUSSION THREAD

**Discussion by MFK2021** [awslabs/gluonts#1774] [Q&A]
Hi, I am trying to train multiple models on a very big dataset (over 100 of different products across 6 different retailers). I have weekly data since the beginning of 2019. When I am training the models on only one product, everything works fine but as soon as I include more than one product I get the error message. Could it be a problem regarding the timestamp I computed for the data? 
Thank you! 

<img width="1025" alt="Screenshot 2021-11-18 at 14 53 30" src="https://user-images.githubusercontent.com/94365082/142440291-ebffde5e-ef25-4e4f-a028-0c1e55ae0a0f.png">

**Comment by mbohlkeschneider** [comment]
Hi @MFK2021,

thank you for raising this issue. It is hard to tell without a code example that produces the error. Could you add some synthetic data and the code that leads to the error?

**Reply by MFK2021** [reply]
My training set is quite big, thats probably why I am getting this error. I just do not know how to solve it. I'll get this error as soon as I am training the estimator (SimpleFeedForward in this case but I also tried it with other models). Maybe this screenshot helps: 

<img width="706" alt="Screenshot 2021-11-18 at 20 25 11" src="https://user-images.githubusercontent.com/94365082/142491376-5a243fb7-4229-43a7-8587-cc7739e6655e.png">
